<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableWidget</class>
 <widget class="QMainWindow" name="tableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Dashboard</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>GPS</string>
      </property>
      <widget class="QLabel" name="LABEL_UTCTIME">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>UTC Time:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_LATITUDE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Latitude:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_LONGITUDE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Longitude:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_ELEVATION">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Elevation:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_GPSCONNECTED">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 193);</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_GPSFIX">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 193);</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_UTCOFFSET">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>UTC Offset:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>NETWORK</string>
      </property>
      <widget class="QLabel" name="LABEL_SERVERSTATUS_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_SERVERSTATUS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(193, 193, 193);</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="LABEL_IPINFO">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>IMAGERS</string>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>MAIN</string>
       </property>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>LINK UP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>STREAMING</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>GUIDE</string>
       </property>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>LINK UP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>STREAMING</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>RECEIVERS</string>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>L-BAND</string>
       </property>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>LINK UP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>STREAMING</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>S-BAND</string>
       </property>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>LINK UP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LABEL_GPSCONNECTED_10">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(193, 193, 193);</string>
        </property>
        <property name="text">
         <string>STREAMING</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Raw driver data</string>
     </attribute>
     <widget class="QTreeView" name="treeview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_EXPANDALL">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>expand all</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_ENABLESIM">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>enable sim</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_DISABLESIM">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>disable sim</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_DISCONNECT">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>DISCONNECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_CONNECT">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BUTTON_COLLAPSEALL">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>collapse all</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="LOGGING_MESSAGE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>791</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Logging</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
